src/operator/nn/mkldnn/mkldnn_base.cc

```diff
@@ -553,8 +553,9 @@ void OpCheck::Run(mxnet::FCompute fn, const nnvm::NodeAttrs &attrs,
   for (size_t i = 0; i < out_blobs.size(); i++)
     out_blobs[i] = outputs[i].data();
   fn(attrs, ctx, in_blobs, req, out_blobs);
-
+#ifndef NDEBUG
```

Comment:
Thank you for fixing this. How about changing to check the environmental variable `MXNET_MKLDNN_DEBUG`?  We used this variable in other places.

Reply:
@TaoLv sure, will do that.

Reply:
Thank you @larroy . We're thinking about adding verbose levels to `MXNET_MKLDNN_DEBUG` in the future.